### DISCUSSION TITLE
New feature request

### DISCUSSION THREAD

**Discussion by Weltolk** [fraxiinus/ReplayBook#88] [Feature Request]
Download a friend's video

**Comment by fraxiinus** [comment]
I'm not sure what you mean by this. If you want to be able to download other people's replays, I have no idea if that is possible. But there is no video to download.

**Reply by Weltolk** [reply]
> I'm not sure what you mean by this. If you want to be able to download other people's replays, I have no idea if that is possible. But there is no video to download.

ok,thanks
